platformio.ini

```diff
@@ -58,8 +58,10 @@ lib_extra_dirs              =
 ;disable_map_gz = 1
 ; Uncomment and specify a folder where to place the map file(s) (default set to folder build_output)
 ;map_dir = /tmp/map_files/
-; Uncomment if you do NOT want additionally gzipped firmware file(s)
+; Uncomment if you do NOT want additionally gzipped firmware file(s) at all
 ;disable_bin_gz = 1
+; Comment if you want additionally gzipped esp32x firmware file(s)
+disable_esp32_gz = 1

```

Comment:
Can you make this the other way around. I don't want to change anything in platformio.ini so I've copied the [tasmota] part into my platform_override.ini file.

If this is enabled by default here I cannot override.